src/main/java/world/bentobox/bentobox/schems/Clipboard.java

```diff
@@ -257,14 +296,10 @@ private void pasteBlock(World world, Island island, Location location, Configura
         Block block = world.getBlockAt(x, y, z);
         String blockData = config.getString("bd");
         if (blockData != null) {
-            if (config.getBoolean(ATTACHED)) {
```

Comment:
This whole section was not actually ever used and was from an earlier version of schems. So I deleted it.